commcare_connect/opportunity/management/commands/delete_duplicate_visits.py

```diff
@@ -0,0 +1,62 @@
+from django.core.management import BaseCommand
+from django.db import transaction
+from django.db.models import Count
+
+from commcare_connect.opportunity.models import UserVisit
+
+
+class Command(BaseCommand):
+    help = "Clean up duplicate visits."
+
+    def add_arguments(self, parser, *args, **kwargs):
+        parser.add_argument("--opp", type=int, help="Opportunity ID to clean up duplicate visits.")
+        parser.add_argument(
+            "--dry-run",
```

Comment:
nit: dry run should probably be the default for safety reasons, and the added arg should be what causes it to delete (then you can't accidentally delete)

Reply:
It would probably also be nice to add a confirmation prompt when deleting